frontend/src/container/TopNav/DateTimeSelection/config.ts

```diff
@@ -42,8 +42,9 @@ export interface Option {
 }
 
 export const ServiceMapOptions: Option[] = [
-	{ value: '1min', label: 'Last 1 min' },
```

Comment:
One minute is too small and sometimes pre-aggregated data might yet exists. So update to minimum of 5 mins and add few more options.